scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -152,6 +156,9 @@ workflow GvsExtractCallset {
                                           else if effective_scatter_count <= 500 then 17 + extract_overhead_memory_override_gib
                                                else 9 + extract_overhead_memory_override_gib
 
+  # support a custom reference if one is provided.  There's gotta be a cleaner way
+  File extract_reference = if defined(custom_reference) then "~{custom_reference}" else GetReference.reference.reference_fasta
```

Comment:
maybe
```suggestion
  File extract_reference = select_first([custom_reference,  GetReference.reference.reference_fasta])
```

Reply:
Huh. I'm actually shocked that this solved the core issue I was fighting, which was the error `Cannot coerce expression of type 'String?' to 'File'`

Something about the implementation of select_first must allow it to auto-convert without the `"~{}"` nonsense that was previously required. Duly noted!

Reply:
I think it may be that there is an automagic conversion from `String` to `File`, but not from `String?` to `File` since the string may not exist.

Reply:
yeah, that conversion was to make it to String instead of String?.  I like the directly cleaner way of doing it